### DISCUSSION TITLE
The confidence of bbox deformable detr predicts is very slow

### DISCUSSION THREAD

**Discussion by GivanTsai** [open-mmlab/mmdetection#9160] [Q&A] [answered]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmdetection/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmdetection).

### 💬 Describe the reimplementation questions

Deformable detr predicts num_queries bbox with confidence and corrdinates, however, the max score of boxes is always very slow, often below 0.4. How can I increase the score of bboxes?

### Environment

。

**Comment by jbwang1997** [comment]
Did you use the pre-trained model provided in mmdetection?

**Comment by ZwwWayne** [comment, marked as answer]
Because the detector uses Focal loss, it is common that the scores of boxes are low.
